src/Link.tsx

```diff
@@ -19,6 +19,7 @@ export interface LinkOwnProps {
   replace?: boolean;
   target?: string;
   to: string;
+  className: string;
```

Comment:
This would mean that the `className` prop would not be optional.
Do you have a specific reason for requiring it or was this by accident? If not then please change it to `className?`.